dotnet/SimpleSauce.Test/SauceSessionTests.cs

```diff
@@ -118,8 +118,32 @@ private static void AssertUsernameAndAccessKeyExist(Root configuredSauceOptions)
             configuredSauceOptions.SauceOptions.AccessKey.Should().NotBeNullOrEmpty();
         }
         //TODO need a test that will validate that
-        //DriverImplementation.CreateRemoteWebDriver(Options.ConfiguredSafariOptions);
+        //Driver.CreateRemoteWebDriver(Options.ConfiguredSafariOptions);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4739713)